src/analytics/ValoraAnalytics.ts

```diff
@@ -108,7 +108,8 @@ class ValoraAnalytics {
         : null
 
       await Statsig.initialize(STATSIG_API_KEY, stasigUser, {
-        overrideStableID: uniqueID, //Received an error if Stable ID not manually specified
+        // StableID should match Segment anonymousId
+        overrideStableID: await Analytics.getAnonymousId(),
```

Comment:
good find.

worth noting that this means there will be a backwards-incompatible change to the data flowing through Statsig, meaning the data that goes from Statsig to BQ is going to have a breaking change. 

Can you let Gowtham know? And can you deal with the fallout from the breaking change? We probly need to copy the data we have into something that's clearly named to indicate it's old/incompatible, clear out the statsig tables in BQ and start fresh. Lmk if you want to discuss.

Reply:
I let him know and setup the tables in BQ.